pkg/lib/cockpit-components-plot.scss

```diff
@@ -0,0 +1,36 @@
+.ct-plot {
+    // Ensure there's no vertical-align offset (set in PF4, for icons)
+    vertical-align: 0;
+
+    &-title {
+        font-size: var(--pf-global--FontSize--md);
+    }
+
+    // Placeholder string to stretch the column, set offscreen
+    &-widest {
+        fill: transparent;
+    }
+
+    &-unit {
+        fill: var(--pf-global--Color--200);
+    }
+
+    &-axis,
+    &-unit {
+        font-size: var(--pf-global--FontSize--xs);
+        fill: var(--pf-global--Color--200);
+    }
+
+    &-lines,
+    &-ticks {
+        stroke: var(--pf-chart-global--Fill--Color--300, var(--pf-global--Color--light-300));
```

Comment:
I was hesitant to make a hard dependency on PF Charts. After looking at the file, I think it's OK, as it just assigns variables (which is what we want).

I can redo the colors to be the chart ones, so our SVG Plots are always up to date with the correct colors, font sizes, set.

Reply:
What is the issue on relying on patternfly-charts.scss variables? Or the reasoning behind 'I was hesitant to make a hard dependency on PF Charts.'.